tools/rstlint.py

```diff
@@ -111,12 +197,14 @@ def check_line_length(fn, lines):
     for lno, line in enumerate(lines):
         if len(line) > 81:
             # don't complain about tables, links and function signatures
-            if line.lstrip()[0] not in '+|' and \
-               'http://' not in line and \
-               not line.lstrip().startswith(('.. function',
-                                             '.. method',
-                                             '.. cfunction')):
-                yield lno+1, "line too long"
+            if (
+                line.lstrip()[0] not in '+|'
+                and 'http://' not in line
+                and not line.lstrip().startswith(
+                    ('.. function', '.. method', '.. cfunction')
```

Comment:
The current formatting of the `startswith()` argument looks more readable to me.

Reply:
For this one, I prefer the revision.

Reply:
This one feels like it could go either way. If it's cool with folks, I'm going to err on the side of a new contributor and keep the revision which puts all of the options for starting lines in one tuple.

Reply:
For ``startswith`` both styles are ok with me (with a slight preference about the new one.
However I wouldn't go on a newline after the first ``(`` and before the last ``):``.

Reply:
This style of bracket indentation is a core tenet of Black. It indents contents of all brackets the same. That makes it symmetrical and provides some usability features:
- for `def func(`, indenting after the opening bracket decouples contents from the length of the function name
- putting `):` on its own dedented line provides a visual delimiter between the function signature and body, or between the `if` statement test and body, or `while` condition and body, and so on.